README.md

```diff
@@ -1266,104 +1751,20 @@
   - It's okay to create get() and set() functions, but be consistent.
 
     ```javascript
-    function Jedi(options) {
-      options || (options = {});
-      var lightsaber = options.lightsaber || 'blue';
-      this.set('lightsaber', lightsaber);
-    }
-
-    Jedi.prototype.set = function(key, val) {
-      this[key] = val;
-    };
-
-    Jedi.prototype.get = function(key) {
-      return this[key];
-    };
-    ```
-
-**[⬆ back to top](#table-of-contents)**
-
-
-## Constructors
-
-  - Assign methods to the prototype object, instead of overwriting the prototype with a new object. Overwriting the prototype makes inheritance impossible: by resetting the prototype you'll overwrite the base!
-
-    ```javascript
-    function Jedi() {
-      console.log('new jedi');
-    }
-
-    // bad
-    Jedi.prototype = {
-      fight: function fight() {
-        console.log('fighting');
-      },
-
-      block: function block() {
-        console.log('blocking');
+    class Jedi {
+      constructor(options = {}) {
+        const lightsaber = options.lightsaber || 'blue';
+        this.set('lightsaber', lightsaber);
```

Comment:
They predate ES6 (ES5.5?). They're g2g in [IE9+](http://robertnyman.com/javascript/javascript-getters-setters.html#regular-getters-and-setters). [Babel compiles](https://babeljs.io/repl/#?experimental=false&evaluate=true&loose=false&spec=false&code=class%20Jedi%20%7B%0A%20%20constructor(options%20%3D%20%7B%7D)%20%7B%0A%20%20%20%20this.lightsaber%20%3D%20options.lightsaber%20%7C%7C%20'blue'%3B%0A%20%20%7D%0A%0A%20%20get%20lightsaber()%20%7B%0A%20%20%20%20return%20(this.team%20%3D%3D%3D%20'rebel')%20%3F%20'blue'%20%3A%20'red'%3B%0A%20%20%7D%0A%0A%20%20set%20lightsaber(ls)%20%7B%0A%20%20%20%20this.team%20%3D%20(ls%20%3D%3D%3D%20'blue')%20%3F%20'rebel'%20%3A%20'empire'%3B%0A%20%20%7D%0A%7D) them to `Object.defineProperty` calls since it looks like IE8 doesn't support the `get` and `set` syntax.